go.mod

```diff
@@ -1,6 +1,6 @@
 module oras.land/oras
 
-go 1.22.0
```

Comment:
We don't need to update the minimum go version in go mod, do we?

Reply:
You mean patch version? Tried `go mod init test` and it gives me patch version go.mod
```go.mod
module test

go 1.22.1
```

Reply:
So yes we should update go patch version.